tests/BotTest.php

```diff
@@ -781,3 +804,25 @@ public function handleEvent()
         // left empty for stubbing
     }
 }
+
+/**
+ * Plugin class which implements PluginContainerInterface.
+ */
+class TestPluginContainer extends AbstractPlugin implements PluginContainerInterface
```

Comment:
Point. I've filed mlively/Phake#185 for the addition of that feature to Phake to be explored.